site/content/contribute/mobile/developer-setup/_index.md

```diff
@@ -79,14 +79,22 @@ $ npm -g install react-native-cli
 #### Install Ruby
 ##### Windows 10
 - Install Ruby from [here](https://rubyinstaller.org/)
+
+##### MacOs
+
+While MacOS supplies a Ruby environment, it's being deprecated and it will only suffice if your architecture is Intel-based. For M1-based architecture, it's recommended to install Ruby using Homebrew.
```

Comment:
What about rvm? I use rvm with no issues but never tried it on an M1 chipset

Reply:
I haven't tried that, so I don't know if that works. brew was the recommended approach where i looked

Reply:
Look into rvm and add it as an option if it works, the same way we do for node

Reply:
it will have to wait then, I'm not at my place and I didn't carry the mac mini around and my laptop is intel based. I'll be back in 3 weeks so I'll take a look after I'm back. Thanks for clarifying